Formula/redis.rb

```diff
@@ -1,8 +1,8 @@
 class Redis < Formula
   desc "Persistent key-value database, with built-in net interface"
   homepage "https://redis.io/"
-  url "http://download.redis.io/releases/redis-4.0.4.tar.gz"
-  sha256 "35768145335e874b1b810e23494ad3daa6f442c3dc1d7e3784992ba50799c0cd"
+  url "https://github.com/antirez/redis/archive/4.0.5.tar.gz"
```

Comment:
This should continue to use the canonical URL

Reply:
I was under the impression that `Homebrew` prefers `HTTPS` URLs over plain `HTTP`, when available from the upstream project maintainer. Anyway, I have changed it back to the conventional URL.

Reply:
Yep, but canonical http over non-canonical https